files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -22,21 +20,13 @@ Builds a repository
 .DESCRIPTION
 Invokes the default MSBuild lifecycle on a repostory. This will download any required tools.
 
-.PARAMETER Path
-The path to the repository to be compiled
-
 .PARAMETER MSBuildArgs
 Arguments to be passed to the main MSBuild invocation
 
 .EXAMPLE
-Invoke-RepositoryBuild $PSScriptRoot /p:Configuration=Release /t:Verify
-
-.NOTES
-This is the main function used by most repos.
+Invoke-RepositoryBuild /p:Configuration=Release /t:Verify
 #>
 function Invoke-RepositoryBuild(
-    [Parameter(Mandatory = $true)]
```

Comment:
We can make life easier on ourselves if we keep this and `Install-Tools` backwards compatible. Make the parameters that were mandator optional, and fallback to `$global:KoreBuildSettings`. 

Otherwise, you have to update every single build.ps1/sh script out there are the same time to prevent breaking.